### DISCUSSION TITLE
Next Release

### DISCUSSION THREAD

**Discussion by mrberman87** [SpinalHDL/SpinalHDL#1425] [General]
When do you think the next release will be?

**Comment by Dolu1990** [comment]
As soon as somebody ask for it, or there is a major feature added.

Do you ask for a release ^^ ?

**Reply by Readon** [reply]
Before that I think #1418 should be considered, the original interface for usage is not good IMO.

**Reply by mrberman87** [reply]
Yes, I would like a couple of the features since last release in my development environment.  I have no problem waiting for that feature to be finished.

**Reply by jijingg** [reply]
Just give me a couple of days, I'll refine the test cases for Regif, The refactoring of regif has been completed, and I would like to merge it in this version

**Reply by mrberman87** [reply]
I am more than happy to wait for the regif updates!

**Comment by Dolu1990** [comment]
Are we all set for a release ? @Readon @jijingg

**Reply by Readon** [reply]
to  me it's ok

**Comment by Dolu1990** [comment]
SpinalHDL 1.10.2 released
